.ci_support/linux_64_variantall.yaml

```diff
@@ -13,19 +13,16 @@ cxx_compiler:
 cxx_compiler_version:
 - '13'
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
 fontconfig:
 - '2'
 freetype:
 - '2'
 occt:
-- 7.7.2
+- 7.8.1
 target_platform:
```

Comment:
Any ideas why occt 7.8.1 is listed here?

Reply:
That's a consequence of re-rendering the recipe.

Reply:
But why should be occt be listed there, doesn't make sense to me.

Reply:
As mentioned above, those files are not controlled by the recipe. 7.8.1 is the current global pin (see [here](https://conda-forge.org/docs/maintainer/pinning_deps/#globally-pinned-packages) and [here](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/27a19a57738536b68039e6a6247de3974fa43a04/recipe/conda_build_config.yaml#L758) ). How conda-forge works is not in the scope of this PR, what is the concern here?

Reply:
it's the same for the vtk-feedstock. I will create a new issue for conda-forge. It's just strange that this pin is included here.

Reply:
https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/7142